src/subcommand/primers_main.cpp

```diff
@@ -21,7 +21,23 @@ void help_primers(char** argv) {
          << "    -z, --zero-variance        allow no variance in the product" << endl
```

Comment:
Does this mean zero variation? If so, that's not the same thing as variance. Variance is a property of a random variable.